examples/data-sources/hpegl_vmaas_tier1_router/data-source.tf

```diff
@@ -0,0 +1,5 @@
+# (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
+data "hpegl_vmaas_tier1_router" "tier1_router" {
```

Comment:
is there a data source exclusively for hpegl_vmaas_tier1_router ? I don't think so. you can add under **https://github.com/HewlettPackard/hpegl-vmaas-terraform-resources/tree/master/examples/data-sources/hpegl_vmaas_router** and change the name to nsx-tier0 and nsx-tier1 for the file names.